### PR TITLE
Update config.toml

### PR DESCRIPTION
**Pull request by Xlabjun**
.cargo/config.toml[C 16:45:17.660 NotebookApp]

    To access the notebook, open this file in a browser:
        file:///home/jovyan/.local/share/jupyter/runtime/nbserver-473-open.html
    Or copy and paste one of these URLs:
        http://(a53c675705c1 or 127.0.0.1):8888/?token=c6338ab860bb6cdf7f976a4676b6a2a8d2e3dc84107a0abd16:45jovyan@a53c675705c1:/workspaces/Quantum/$ jupyter notebook --ip=0.0.0.0127.0.0.1host.pyjovyan@a53c675705c1:/workspaces/Quantum$ cd Samples/src/Teleportation/
jovyan@a53c675705c1:/workspaces/Quantum/Samples/src/Teleportation$ dotnet run
Round 0:        Sent True,      got True.
Teleportation successful!!

Round 1:        Sent True,      got True.
Teleportation successful!!

Round 2:        Sent False,     got False.
Teleportation successful!!

Round 3:        Sent False,     got False.
Teleportation successful!!

Round 4:        Sent True,      got True.
Teleportation successful!!

Round 5:        Sent False,     got False.
Teleportation successful!!

Round 6:        Sent True,      got True.
Teleportation successful!!

Round 7:        Sent True,      got True.
Teleportation successful!!

jovyan@a53c675705c1:/workspaces/Quantum/Samples/src/Teleportation$ python host.py
Preparing Q# environment...
['Microsoft.Quantum.Samples.Teleportation.IsMinus', 'Microsoft.Quantum.Samples.Teleportation.IsPlus', 'Microsoft.Quantum.Samples.Teleportation.PrepareRandomMessage', 'Microsoft.Quantum.Samples.Teleportation.SetToMinus', 'Microsoft.Quantum.Samples.Teleportation.SetToPlus', 'Microsoft.Quantum.Samples.Teleportation.Teleport', 'Microsoft.Quantum.Samples.Teleportation.TeleportClassicalMessage', 'Microsoft.Quantum.Samples.Teleportation.TeleportRandomMessage']
Sending |->
Received |->
------------------
Sent True, Received: True
Sent False, Received: False
------------------
Estimated resources needed for teleport:
 {'CNOT': 2, 'QubitClifford': 4, 'R': 0, 'Measure': 8, 'T': 0, 'Depth': 0, 'Width': 3, 'BorrowedWidth': 0}
------------------
Sending |+>
Received |+>
------------------
Sending |+>
Received |+>
------------------
Sending |->
Received |->
------------------
Sending |+>
Received |+>
------------------
Sending |+>
Received |+>
------------------
Sending |->
Received |->
------------------
Sending |->
Received |->
------------------
Sending |+>
Received |+>
------------------
Sending |->
Received |->
------------------
Sending |+>
Received |+>
------------------.devcontainer

### CODE REVIEW

**Comment by Xlabjun** [comment]
> .cargo/config.toml[C 16:45:17.660 NotebookApp]
> 
> ```
> To access the notebook, open this file in a browser:
>     file:///home/jovyan/.local/share/jupyter/runtime/nbserver-473-open.html
> Or copy and paste one of these URLs:
>     http://(a53c675705c1 or 127.0.0.1):8888/?token=c6338ab860bb6cdf7f976a4676b6a2a8d2e3dc84107a0abd16:45jovyan@a53c675705c1:/workspaces/Quantum/$ jupyter notebook --ip=0.0.0.0127.0.0.1host.pyjovyan@a53c675705c1:/workspaces/Quantum$ cd Samples/src/Teleportation/
> ```
> 
> jovyan@a53c675705c1:/workspaces/Quantum/Samples/src/Teleportation$ dotnet run Round 0: Sent True, got True. Teleportation successful!!
> 
> Round 1: Sent True, got True. Teleportation successful!!
> 
> Round 2: Sent False, got False. Teleportation successful!!
> 
> Round 3: Sent False, got False. Teleportation successful!!
> 
> Round 4: Sent True, got True. Teleportation successful!!
> 
> Round 5: Sent False, got False. Teleportation successful!!
> 
> Round 6: Sent True, got True. Teleportation successful!!
> 
> Round 7: Sent True, got True. Teleportation successful!!
> 
> ## jovyan@a53c675705c1:/workspaces/Quantum/Samples/src/Teleportation$ python host.py
> Preparing Q# environment...
> ['Microsoft.Quantum.Samples.Teleportation.IsMinus', 'Microsoft.Quantum.Samples.Teleportation.IsPlus', 'Microsoft.Quantum.Samples.Teleportation.PrepareRandomMessage', 'Microsoft.Quantum.Samples.Teleportation.SetToMinus', 'Microsoft.Quantum.Samples.Teleportation.SetToPlus', 'Microsoft.Quantum.Samples.Teleportation.Teleport', 'Microsoft.Quantum.Samples.Teleportation.TeleportClassicalMessage', 'Microsoft.Quantum.Samples.Teleportation.TeleportRandomMessage']
> Sending |->
> Received |->
> ## Sent True, Received: True
> Sent False, Received: False
> ## Estimated resources needed for teleport:
> {'CNOT': 2, 'QubitClifford': 4, 'R': 0, 'Measure': 8, 'T': 0, 'Depth': 0, 'Width': 3, 'BorrowedWidth': 0}
> ## Sending |+>
> Received |+>
> ## Sending |+>
> Received |+>
> ## Sending |->
> Received |->
> ## Sending |+>
> Received |+>
> ## Sending |+>
> Received |+>
> ## Sending |->
> Received |->
> ## Sending |->
> Received |->
> ## Sending |+>
> Received |+>
> ## Sending |->
> Received |->
> Sending |+> Received |+> ------------------.devcontainer

To access the notebook, open this file in a browser:
    file:///home/jovyan/.local/share/jupyter/runtime/nbserver-473-open.html
Or copy and paste one of these URLs:
    http://(a53c675705c1 or 127.0.0.1):8888/?token=c6338ab860bb6cdf7f976a4676b6a2a8d2e3dc84107a0abd16:45jovyan@a53c675705c1:/workspaces/Quantum/$ jupyter notebook --ip=0.0.0.0127.0.0.1host.pyjovyan@a53c675705c1:/workspaces/Quantum$ cd Samples/src/Teleportation/127.0.0.1[C 16:45:17.660 NotebookApp]

    To access the notebook, open this file in a browser:
        file:///home/jovyan/.local/share/jupyter/runtime/nbserver-473-open.html
    Or copy and paste one of these URLs:
        http://(a53c675705c1 or 127.0.0.1):8888/?token=c6338ab860bb6cdf7f976a4676b6a2a8d2e3dc84107a0abdXlabjun:patch-1.cargo/config.toml16:45jovyan@a53c675705c1:/workspaces/Quantum/$ jupyter notebook --ip=0.0.0.0

**Review by Xlabjun — COMMENTED** [review]
To access the notebook, open this file in a browser:
    file:///home/jovyan/.local/share/jupyter/runtime/nbserver-473-open.html
Or copy and paste one of these URLs:
    http://(a53c675705c1 or 127.0.0.1):8888/?token=c6338ab860bb6cdf7f976a4676b6a2a8d2e3dc84107a0abd16:45jovyan@a53c675705c1:/workspaces/Quantum/$ jupyter notebook --ip=0.0.0.0127.0.0.1host.pyjovyan@a53c675705c1:/workspaces/Quantum$ cd Samples/src/Teleportation/

**Review by Xlabjun — COMMENTED** [review]
To access the notebook, open this file in a browser:
    file:///home/jovyan/.local/share/jupyter/runtime/nbserver-473-open.html
Or copy and paste one of these URLs:
    http://(a53c675705c1 or 127.0.0.1):8888/?token=c6338ab860bb6cdf7f976a4676b6a2a8d2e3dc84107a0abd16:45jovyan@a53c675705c1:/workspaces/Quantum/$ jupyter notebook --ip=0.0.0.0127.0.0.1host.pyjovyan@a53c675705c1:/workspaces/Quantum$ cd Samples/src/Teleportation/

File: .cargo/config.toml
Comment (Xlabjun) [inline .cargo/config.toml:1]: To access the notebook, open this file in a browser:
file:///home/jovyan/.local/share/jupyter/runtime/nbserver-473-open.html

**Review by Xlabjun — COMMENTED** [review]
To access the notebook, open this file in a browser:
file:///home/jovyan/.local/share/jupyter/runtime/nbTo access the notebook, open this file in a browser:
    file:///home/jovyan/.local/share/jupyter/runtime/nbserver-473-open.html
Or copy and paste one of these URLs:
    http://(a53c675705c1 or 127.0.0.1):8888/?token=c6338ab860bb6cdf7f976a4676b6a2a8d2e3dc84107a0abd16:45jovyan@a53c675705c1:/workspaces/Quantum/$ jupyter notebook --ip=0.0.0.0127.0.0.1host.pyjovyan@a53c675705c1:/workspaces/Quantum$ cd Samples/src/Teleportation/Xlabjun:patch-1server-473-open.html2056

**Review by swernli — CHANGES_REQUESTED** [review]
This change breaks Rust builds due to invalid format of the toml file. Further, the PR description, comments, and linked issue appear to be auto-generated from some tooling and do not describe the intent of the change. I'll close this and the related issue for now; please reopen or file again with clearer information.

**Pull request closed without merging**